frontend/src/component/releases/ReleasePlanTemplate/CreateReleasePlanTemplate.tsx

```diff
@@ -29,12 +35,21 @@ const StyledCancelButton = styled(Button)(({ theme }) => ({
     marginLeft: theme.spacing(3),
 }));
 
+const StyledAddMilestoneButton = styled(Button)(({ theme }) => ({
+    marginTop: theme.spacing(1),
+    maxWidth: theme.spacing(20),
+}));
+
 export const CreateReleasePlanTemplate = () => {
     const releasePlansEnabled = useUiFlag('releasePlans');
     usePageTitle('Create release plan template');
     const { setToastApiError, setToastData } = useToast();
     const navigate = useNavigate();
     const { createReleasePlanTemplate } = useReleasePlanTemplatesApi();
+    const [milestones, setMilestones] = useState<
```

Comment:
Why are we using a new state instead of using the one from `useTemplateForm`?

Reply:
it's a weird thing. I tried that but I kept getting rerenders/losing them again immediately after adding when using the `useTemplateForm`, this was a way to get across the finish line